app/src/main/java/com/uncookthebook/app/ServerManager.java

```diff
@@ -0,0 +1,40 @@
+package com.uncookthebook.app;
+
+import android.util.Log;
+
+import com.google.android.gms.auth.api.signin.GoogleSignInAccount;
+import com.uncookthebook.app.models.TokenizedObject;
+import com.uncookthebook.app.models.User;
+import com.uncookthebook.app.network.APIService;
+import com.uncookthebook.app.network.APIServiceUtils;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+
+class ServerManager {
```

Comment:
I don't think we have to put the calls to the server to a different class. We should put the code when it's needed, otherwise we lose the possibility to modify the behavior of onResponse and onFailure